tests/data/mstool/input/DiagnosticReport.ndjson

```diff
@@ -0,0 +1,24 @@
+{
+  "resourceType" : "DiagnosticReport",
+  "identifier" : [{ "value": "dropped" }],
+  "basedOn" : [{ "reference": "CarePlan/x" }],
+  "status" : "final",
+  "category" : [{ "text": "kept" }],
+  "code" : { "text": "kept" },
+  "subject" : { "reference": "Patient/x" },
+  "encounter" : { "reference": "Encounter/x" },
+  "effectiveDateTime" : "2023",
```

Comment:
For "variant" fields, I don't usually unit-test each possible variation. Only the "interesting" variations (like if we drop the string variant, but keep the others, I'll have a unit test for an arbitrary one that we keep and also a unit test for the dropped variant).

We can't put multiple variants in one test file because the MS tool would complain about invalid FHIR.